client/reader/sidebar/index.jsx

```diff
@@ -141,7 +141,14 @@ export class ReaderSidebar extends React.Component {
 								} ) }
 							>
 								<a href="/" onClick={ this.handleReaderSidebarFollowedSitesClicked }>
-									<Gridicon icon="checkmark-circle" size={ 24 } />
+									<svg
```

Comment:
should all of these be in the material-icons package?

Reply:
I tried adding the icons to the package but hit a wall with @griffbrad — He mentioned adding all the icons to the package, rather than doing it icon-by-icon. So, rather than wait for that to happen, I figured it wouldn't hurt to replace the Gridicons with SVGs. Then, once the package is updated with all the icons, I can go back and use the component.

Reply:
How about packaging these icons separately instead of inlining them? Something like:

file: material-icons.jsx
```
const materialIcons = {
	'chat': (
		<svg
				className="material-icon mi_chat"
				xmlns="http://www.w3.org/2000/svg"
				width="24"
				height="24"
			>
			<path d="M4 4h16v12H5.17L4 17.17V4m0-2c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2H4zm2 10h8v2H6v-2zm0-3h12v2H6V9zm0-3h12v2H6V6z" />
		</svg>
	),
	...
}

export function MaterialIcon( { icon } ) {
	return materialIcons[icon];
}
```

Reply:
Unless I'm missing something, I don't think its worth the effort. We already have the `material-design-icons` package, but its busted (for me). Dropping the SVGs is just a quicker way to get this resolved while the package is awaiting updates.

Reply:
> Unless I'm missing something, I don't think its worth the effort. We already have the `material-design-icons` package, but its busted (for me). Dropping the SVGs is just a quicker way to get this resolved while the package is awaiting updates.

Ah, I didn't know about `material-design-icons`. I was able to update the package and make it work. Here is the result.

<img width="271" alt="screenshot_714" src="https://user-images.githubusercontent.com/127594/59539614-cc095f80-8eb2-11e9-9ec5-c0e21ef25932.png">

Instructions on how to add icons wasn't clear so I've updated that as well.